package.json

```diff
@@ -18,10 +18,7 @@
     "collectCoverage": true,
     "collectCoverageFrom": [
       "src/**/*.js"
-    ],
-    "transform": {
-      "^.+\\.js$": "<rootDir>/node_modules/react-native/jest/preprocessor.js"
```

Comment:
The `preprocessor.js` was deprecated, If I remove it `jest` complains about some files not being transformed so I added the `babel.config.js` to transform those files 👀